homeassistant/components/scene/services.yaml

```diff
@@ -20,3 +20,17 @@ apply:
         light.ceiling:
           state: "on"
           brightness: 80
+
+create:
+  description: Creates a new scene.
+  fields:
+    scene_id:
+      description: The entity_id of the new scene.
+      example: all_lights
+    entities:
+      description: The entities to control with the scene.
+      example:
```

Comment:
This is not very nice. Any better ideas?
![grafik](https://user-images.githubusercontent.com/15657682/67708774-03d5f700-f9bd-11e9-8192-85b6d9664e17.png)

Reply:
@bramkragten is planning on making a full featured UI for this. We should also allow better representing the example data.

Reply:
Great. So I'll leave it like this